### ISSUE TITLE
FXManifest not formatted correctly, and MYSQL-ASync errors

### ISSUE THREAD

**Issue by Bobertsawesome** [Tazi0/esx_invest#3]
Hello there, I just wanted to share the issues I am experiencing with your esx_invest script.

First, the FXManifest file is missing the fx_version and game setting. Will not load without it (it was still formatted in the old __resource.lua, even though the filename was changed).

Second, the script is unfunctional on startup, it returns a script error for every script running, along the lines of this: [Gyazo](https://gyazo.com/dea5cf0994d2d4eca72a1d3359e0f87e)

Also, the UI greeting page "please place thumb..." pops up immediately when a client joins or the resource is restarted, never going away unless the resource is stopped entirely. I have updated to your newest version of new_banking, both scripts are in the [esx] folder, too.

If you need further testing just let me know.

Thanks,
Bobert.

**Comment by Bobertsawesome** [comment]
Also, I should note that I know it's not done. Just providing feedback :). Thanks.

**Comment by Tazi0** [comment]
@Bobertsawesome 👋 thanks for the feedback, let me elaborate on what's happening:

1. I'm working on the UI and for me to see the page in my web browser I need to enable the waiting screen right as it starts. So that's why it pops up when you start it.

2. I changed `__resource.lua` to `fxmanifest.lua` without specifying the game it's launched in. This will come in my next commit with some few other updates and stuff

3. About the MySQL `execute` error idk what that issue is since I also got it but didn't see any issue with my code.

Have a good day :)
